docs/tracing.md

```diff
@@ -48,11 +48,13 @@ clippy-tracing \
   --path ./src \
   --exclude \
     benches,\
-    virtio/gen,bindings.rs,net/gen,\
-    log-instrument-macros/,log-instrument/,clippy-tracing/,\
-    vmm_config/logger.rs,logger/,signal_handler.rs,time.rs
+virtio/gen,bindings.rs,net/gen,\
+log-instrument-macros/,log-instrument/,clippy-tracing/,\
+vmm_config/logger.rs,logger/,signal_handler.rs,time.rs
 ```
 
+Whitespace given to the `--exclude` argument is significant.
```

Comment:
meaning "an absence of whitespaces is intentional"? I guess there may be a better way to word that.

Reply:
I think going any deeper requires putting the old format and explaining why it doesn't work, this seems to me a bit much.

Reply:
My concern here is someone will be looking for a whitespace in the `exclude` argument because the sentence implies its existence, but fail to find it. This might be my own way of reading that though. So feel free to ignore if you don't feel the same.

Reply:
I've updated it to be
```
clippy-tracing \
  --action fix \
  --path ./src \
  --exclude benches \
  --exclude virtio/gen,bindings.rs,net/gen \
  --exclude log-instrument-macros/,log-instrument/,clippy-tracing/ \
  --exclude vmm_config/logger.rs,logger/,signal_handler.rs,time.rs
```
does this resolve your concerns?